config/initializers/wrap_parameters.rb

```diff
@@ -7,7 +7,7 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json]
+  wrap_parameters format: []
```

Comment:
Before this change, we would always get an extra param sent to the controller named after the resource (i.e. `params[:profile]`). That doesn't fit with our JSON spec where everything is wrapped in `params[:data]`.